www/include/home/customViews/index.ihtml

```diff
@@ -582,7 +582,8 @@ function toggleEdit(show) {
       if (widgetTitle == 'Click to edit' || widgetTitle == '') {
         jQuery(this).hide();
```

Comment:
@loiclau we could also remove the `widgetTitle == ''` condition, so that, to hide the title bar, we would simply have to let the "Click to edit" default title.
If this suits your needs, I can push this modification.
Thx 👍